app/service.js

```diff
@@ -134,6 +144,9 @@ app.get('/__version__', function(req, res) {
   res.json(versionData);
 });
 
+// The error handler must be before any other error middleware
+app.use(raven.middleware.express.errorHandler(sentryDSN));
```

Comment:
@pdehaan I just spent an hour learning that in express.js, the order in which you specify your middlewares and route handlers is magically significant in hidden and weird ways and in fact this is the ONLY order which is suitable, which is ludicrous and absurd.

![iwkad22](https://cloud.githubusercontent.com/assets/119884/17873486/beefdc5e-6893-11e6-914d-16db1d3b3769.jpg)

As far as having a missing sentry dsn, it will fail silently and carry on for local testing.